src/libotutil/otutil.h

```diff
@@ -26,6 +26,13 @@
 #include <string.h> /* Yeah...let's just do that here. */
 #include <libglnx.h>
 
+/* https://bugzilla.gnome.org/show_bug.cgi?id=766370 */
+#if !GLIB_CHECK_VERSION(2, 50, 0)
```

Comment:
Fixed, though in practice I don't think anyone is going to be using random pre-release snapshots of GLib *after* the release.